### DISCUSSION TITLE
What would a heliologist study?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1167] [Galaxy Trivia]
**Bonus:** What is a mangetout?

**Comment by Camarota-234** [comment]
Geologists are scientists who study the Earth, so a heliologist would be a scientist who studies the Sun.

Also known as the snow or sugar pea, mangetout are a flat-podded variety of pea.

**Comment by peterbaccaro** [comment]
Heliology is the science of Helios, Sol, otherwise known as the **Sun**.

The French name tells you everything; it means 'eat it all'.  A mangetout is a **type of garden pea picked very young, so young that the pod is still flat and the peas have barely developed**.
